examples/ingress-resources/path-regex/README.md

```diff
@@ -189,3 +191,305 @@ Corresponding NGINX config file snippet:
     status_zone "coffee-svc";
 ...
 ```
+
+## Path-Regex Annotation in Mergeable Ingress Types
+
+### Example 1: Path-Regex on Master Ingress
+
+In the following example you enable path regex annotation ``nginx.org/path-regex`` on **Master Ingress**
+and set its value to `case_sensitive` match. A `path-regex` annotation is not explicitely set on **Minion Ingresses**.
+
+Each minion without `path-regex` annotation will inherit it from the master.
+
+Content of `cafe-master.yaml`:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: cafe-ingress-master
+  annotations:
+    nginx.org/mergeable-ingress-type: "master"
+    nginx.org/path-regex: "case_sensitive"
+spec:
+  ingressClassName: nginx
+  tls:
+  - hosts:
+    - cafe.example.com
+    secretName: cafe-secret
+  rules:
+  - host: cafe.example.com
+```
+
+Content of `coffee-minion.yaml`:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: cafe-ingress-coffee-minion
+  annotations:
+    nginx.org/mergeable-ingress-type: "minion"
```

Comment:
I see that path-regex is set at the master but not at the minion.
Therefore I would not expect the minion paths to support regex, only paths defined in the master.
I would expect this path to cause an error.

Reply:
Each ingress is a configuration to itself.  They just get flattened into a single larger server block configuration.

Reply:
@brianehlert we don't support that particular use-case by design, where one have have a master with its own paths on top of minions so imo this behaviour of inheriting master annotation makes sense.
If one tries to add paths to master:
`spec.rules[0].http.paths: Too many: 1: must have at most 0 items`
`Ingress master is invalid or doesn't exist`

Reply:
fixed